railties/lib/rails/generators/rails/app/templates/config/boot.rb.tt

```diff
@@ -1,3 +1,7 @@
+<% unless options.skip_spring? -%>
```

Comment:
I left out [`rescue LoadError`](https://github.com/rails/rails/pull/39225/files#diff-597f6e4e2781654a9c071984ba0940adR4-R6) in favor of `options.skip_spring?`.

Reply:
The currently failing tests clearly rely on the `rescue`, but I think that's only because the call to `bundle exec spring binstub` is stubbed out.  I will look into making those tests pass without the `rescue`.

Reply:
I see now that it's not because `bundle exec spring binstub` is stubbed out, but rather because [`bundle_install?` is false](https://github.com/rails/rails/blob/054b54e8668fa5f8fa656606a9ab1cdafbf58402/railties/lib/rails/generators/app_base.rb#L445).  That seems like an unfortunate circumstance to deal with.  While it's obvious that you must run `bundle install` manually after using `--skip-bundle`, it's not obvious that you must run `bundle exec spring binstub` too.

One way to address this would be to move the binstub template inside of Rails itself.  That doesn't seem too unreasonable since none of the code is particularly specific to Spring (other than `require 'spring/binstub'`).  But that seems out of scope for this PR.  For now, I just added the `rescue`.